rpm/tarantool.spec

```diff
@@ -174,10 +176,14 @@ C and Lua/C modules.
          -DCMAKE_BUILD_TYPE=RelWithDebInfo \
          -DCMAKE_INSTALL_LOCALSTATEDIR:PATH=%{_localstatedir} \
          -DCMAKE_INSTALL_SYSCONFDIR:PATH=%{_sysconfdir} \
-%if %{with backtrace}
-         -DENABLE_BACKTRACE:BOOL=ON \
+%ifnarch aarch64
+    %if %{with backtrace}
+             -DENABLE_BACKTRACE:BOOL=ON \
+    %else
+             -DENABLE_BACKTRACE:BOOL=OFF \
+    %endif
 %else
-         -DENABLE_BACKTRACE:BOOL=OFF \
+    -DENABLE_BACKTRACE:BOOL=OFF \
```

Comment:
It is better to adjust the `with backtrace` condition at the line 60. Don't forget to leave a comment with a link to the relevent issue, where the problem with backtraces is described.

Note: `%bcond_with foo` *disables* foo by default (read it as 'add --with-foo argument for the debuild tool').

Reply:
Changed.

I don't see a relevant issue (#6060 is about M1), so just left a comment.